metrics/config/node-config.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+# this is needed for the scaling scripts
+sudo swupd bundle-add jq
+
+# increase max inotify watchers
+cat <<EOT | sudo bash -c "cat > /etc/sysctl.conf"
+fs.inotify.max_queued_events=1048576
+fs.inotify.max_user_watches=1048576
+fs.inotify.max_user_instances=1048576
+EOT
+sudo sysctl -p
+
+sudo mkdir -p /etc/systemd/system/kubelet.service.d/
+cat <<EOT | sudo bash -c "cat > /etc/systemd/system/kubelet.service.d/limits.conf"
+[Service]
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=1048576
+TimeoutStartSec=0
+MemoryLimit=infinity
+EOT
+
+sudo systemctl daemon-reload
+sudo systemctl restart kubelet
+
+sudo mkdir -p /etc/systemd/system/containerd.service.d/
+cat <<EOT | sudo bash -c "cat > /etc/systemd/system/containerd.service.d/limits.conf"
```

Comment:
It's looking good to me. The only thing is that if we decide or not to move this to the clr-k8s-examples area, we may use the $RUNNER var for the container engine service config, with this we can continue providing diferent container engines support. @krsna1729  @ganeshmaharaj, @dklyle @grahamwhaley,  any thoughts?